packages/docs-site/src/templates/default.js

```diff
@@ -10,6 +10,7 @@ import Footer from '../components/presenters/footer'
 import Layout from '../components/presenters/layout'
 import PostArticle from '../components/presenters/post-article'
 import Sidebar from '../components/presenters/sidebar'
+import CodeHighlighter from '../components/presenters/code-highlighter'
```

Comment:
I know this is added for an example but... I've been working on this file too so we'll get a conflict soon. Just a heads up

Reply:
No worries feel free to just delete the examples out. Tempted to add Storybook into the Gatsby project for the site specific components.